src/harness/reference_models/propagation/ehata/ehata.py

```diff
@@ -86,18 +86,18 @@ def CbsdEffectiveHeights(height_cbsd, its_elev):
   """
 
   npts = int(its_elev[0])
-  xi = its_elev[1] / 1000   # step size of the profile points, in km
+  xi = its_elev[1] / 1000.   # step size of the profile points, in km
   dist_km = npts * xi         # path distance, in km
   elev_cbsd = its_elev[2]
 
   if dist_km < 3.0:
     eff_height = height_cbsd
 
   else: # dist_km >= 3 km
-    i_start = 2 + int(math.ceil(3.0 / xi))
+    i_start = 2 + int(3.0 / xi)
```

Comment:
There's a possibility to start before 3 km

Reply:
Yes it will slightly start before 3km, which is the original NTIA code you can find in the eHata repository:  https://github.com/NTIA/ehata
I remember now that we changed this according to your request at the time, I think.

Seriously this will not significantly change the results, but if you insist we can keep the current implementation. However in that case, please send me a piece of code which fix the code as right now it is incorrect.